### ISSUE TITLE
Add Alert Error View

### ISSUE THREAD

**Issue by zheleznov163** [bitsongofficial/wallet-mobile#116]
Add a global error notification. You need to decide how long the message will last. It's probably best to only display if there are errors in the open bottomsheet, and hide it when trying to collapse the bottomsheet.

<img width="297" alt="Снимок экрана 2022-09-01 в 16 13 04" src="https://user-images.githubusercontent.com/62299367/187922479-4b1c43b9-998d-4268-9cc6-970131da65d4.png">

**Comment by EnricoBarbieri1997** [comment]
I agree that it should close automatically when no bottom sheet is open. Furthermore we should provide a method that accepts the error text or an array of errors and automatically opens it when the array is not empty or the string is not empty and closes it otherwise.

**Comment by EnricoBarbieri1997** [comment]
It's not a bad implementation but if the message is set while no modal is open (maybe an async request started and then the modal was closed by the user) the alert is going to show up anyway. Also the current closing animation feels strange, it fades and then blinks back to fully visible
